bodhi/server/scripts/obsolete_updates_for_archived_releases.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2018 Red Hat, Inc. and Sebastian Wojciechowski
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""This script will change updates status to 'obsolete' for archived releases."""
+
+import click
+from sqlalchemy.sql import or_, and_
+
+from bodhi.server import config, initialize_db, models, Session
+
+@click.command()
+@click.version_option(message='%(version)s')
+def make_updates_obsolete():
+    """Get username SAR data."""
+    initialize_db(config.config)
+    db = Session()
+
+    archived_releases = db.query(models.Release).filter_by(state=models.ReleaseState.archived).all()
+    for release in archived_releases:
+        click.echo("Archived release: {}".format(release.name))
+        updates = db.query(models.Update).filter(models.Update.release_id == release.id).filter(
+                and_(*[models.Update.status != s for s in (
+                    models.UpdateStatus.obsolete, models.UpdateStatus.stable
+                )])
+            ).all()
+        if updates:
+            click.echo("Found {} updates to change".format(len(updates)))
+        else:
+            click.echo("No updates need change.")
+        """
+        for update in updates:
+            update.status = UpdateStatus.obsolete
+            db.commit()
+        """
+
+if __name__ == '__main__':
+    make_updates_obsolete()
```

Comment:
We only need to do this once, when we first deploy the Bodhi that has the auto-obsoletion behavior, so let's do it with a database migration with alembic, instead of a console script.